src/libaktualizr/campaign/campaign.h

```diff
@@ -25,10 +25,15 @@ static inline Cmd cmdFromName(const std::string &name) {
   return std::map<std::string, Cmd>{
       {"campaign_accept", Cmd::Accept}, {"campaign_decline", Cmd::Decline}, {"campaign_postpone", Cmd::Postpone}}
       .at(name);
-}
+};
 
 // Out of uptane concept: update campaign for a device
 class Campaign {
+ public:
+  static std::vector<Campaign> campaignsFromJson(const Json::Value &json);
+  static void JsonFromCampaigns(const std::vector<Campaign> &in, Json::Value &out);
+  static std::vector<Campaign> fetchAvailableCampaigns(HttpInterface &http_client, const std::string &tls_server);
+
```

Comment:
While it seems logical to make those static methods, do we need a namespace for a single class now?

Reply:
I made them static members to make `make docs` (doxygen) happy, for some reason it cannot see these three functions implementation if it is just in a scope of the namespace.

>  do we need a namespace for a single class now?
I think, it can be removed now.